src/_guides/openapi/design-first-laravel-php.md

```diff
@@ -0,0 +1,320 @@
+
+Some folks may be used to the API code-first workflow, where you write the whole API then sprinkle in some metadata later using swagger-php or something similar. The API design-first workflow is the opposite of that approach.
+
+Instead of rushing into the code, we can build OpenAPI descriptions before writing any code at all, like creating a blueprint before building a house. Once you have the OpenAPI description documents saved (ideally in your source code repository), you can leverage it at every step of the API lifecycle, to produce mock APIs for clients to test assumptions with, build client libraries without writing any code, make really effective contract testing, and even generate backend code to get the application teams started once the contract is all signed off.
+
+This guide is going to look at two specific parts of the API design-first workflow, and show how to set it up in Laravel: request validation, and contract testing responses.
+
+- [Getting OpenAPI \& Bump.sh Setup](#getting-openapi--bumpsh-setup)
+- [Request Validation powered by OpenAPI](#request-validation-powered-by-openapi)
+- [Contract Testing with OpenAPI](#contract-testing-with-openapi)
+- [Sample Code](#sample-code)
+
+## Getting OpenAPI & Bump.sh Setup
+
+The API design-first workflow means you'll need to create your OpenAPI description before you start writing all your code, so if you don't have an `openapi.yaml` already that is probably the first step. You can use a wide variety of [graphical editors](https://openapi.tools/#gui-editors), [text editors](https://openapi.tools/#text-editors), or [traffic sniffing](/guides/openapi/code-first/#traffic-sniffing) to generate this OpenAPI, and there is lots of documentation and guides to help you. 
+
+Alternatively you can grab some sample OpenAPI from the API Guru Marketplace, and click JSON or YAML to download their OpenAPI descriptions.
+
+Either way, once you have some an OpenAPI description document, pop it into your Git repository somewhere like `api/openapi.yaml`. 
+
+Building an API for a bunch of clients is always a tricky one, but by deploying the documentation first you can see if people like the look of the API before you waste a bunch of time building it. Then as you progress through, especially if you are adding these tools to an existing codebase, you will continue to find mistakes in your OpenAPI or your actual API code, improving both as you go until you have a perfect match that will never again be broken, solving the "docs vs code" drift problem, and every fix will be deployed to Bump.sh with each commit/merge.
+
+```bash
+$ bump deploy api/openapi.yaml \
+  --doc laravel-design-first \
+  --token my-documentation-token
+
+* Your new documentation version will soon be ready at https://bump.sh/hub/code-samples/doc/laravel-design-first
```

Comment:
This URL gives a 404.

Should we write something like `https://bump.sh/hub/<your-namespace>/doc/laravel-design-first`, instead?